docs/architecture/decisions/0001-media-usage-warning-link.md

```diff
@@ -0,0 +1,20 @@
+# 1. Editors can more clearly see what a change to media will affect
```

Comment:
Remove this file.

Reply:
I will remove. But wanted to make a note that SWS did request that we start adding these ADR documents.

Reply:
Removed.